src/GovUk.Education.ExploreEducationStatistics.Common/Services/FileStoragePathUtils.cs

```diff
@@ -7,96 +8,83 @@ namespace GovUk.Education.ExploreEducationStatistics.Common.Services
+            return $"releases/{releaseId}";
         }
 
-        public static string PublicContentPublicationParentPath(string slug, string prefix = null)
+        public static string PublicContentPublicationParentPath(string slug, string? prefix = null)
         {
             return $"{PublicContentPublicationsPath(prefix)}/{slug}";
         }
 
-        public static string PrivateContentReleaseParentPath(Guid publicationId, Guid releaseId, string prefix = null)
-        {
-            return $"{PrivateContentPublicationParentPath(publicationId, prefix)}/releases/{releaseId}";
-        }
-
-        public static string PublicContentReleaseParentPath(string publicationSlug, string releaseSlug, string prefix = null)
+        public static string PublicContentReleaseParentPath(string publicationSlug,
+            string releaseSlug,
+            string? prefix = null)
         {
             return $"{PublicContentPublicationParentPath(publicationSlug, prefix)}/releases/{releaseSlug}";
         }
 
-        public static string PublicContentPublicationPath(string slug, string prefix = null)
+        public static string PublicContentPublicationPath(string slug, string? prefix = null)
         {
             return $"{PublicContentPublicationParentPath(slug, prefix)}/publication.json";
         }
 
-        public static string PublicContentLatestReleasePath(string slug, string prefix = null)
+        public static string PublicContentLatestReleasePath(string slug, string? prefix = null)
         {
             return $"{PublicContentPublicationParentPath(slug, prefix)}/latest-release.json";
         }
 
-        public static string PublicContentReleasePath(string publicationSlug, string releaseSlug, string prefix = null)
+        public static string PublicContentReleasePath(string publicationSlug, string releaseSlug, string? prefix = null)
         {
             return $"{PublicContentPublicationParentPath(publicationSlug, prefix)}/releases/{releaseSlug}.json";
         }
 
-        public static string PrivateContentDataBlockParentPath(
-            Guid publicationId,
-            Guid releaseId)
+        public static string PublicContentDataBlockParentPath(string publicationSlug, string releaseSlug)
         {
-            return $"{PrivateContentReleaseParentPath(publicationId, releaseId)}/{PublicContentDataBlocksDirectory}";
+            return $"{PublicContentReleaseParentPath(publicationSlug, releaseSlug)}/{DataBlocksDirectory}";
         }
 
-        public static string PublicContentDataBlockParentPath(
-            string publicationSlug,
-            string releaseSlug)
+        public static string PrivateContentDataBlockPath(Guid releaseId, Guid dataBlockId)
         {
-            return $"{PublicContentReleaseParentPath(publicationSlug, releaseSlug)}/{PublicContentDataBlocksDirectory}";
+            return
+                $"{PrivateContentReleaseParentPath(releaseId)}/{DataBlocksDirectory}/{dataBlockId}.json";
```

Comment:
Good point. I think I did some cleanup already to make some of the private paths use less methods, but I've just added an extra commit to inline `"releases/{releaseId}"` too.

Not sure if you spotted it but I also removed the `prefix` attribute from all the private methods since it's only relevant to the public paths where the staging directory might be used.